how-it-works/7_smart_contracts/01_canister-lifecycle.card.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Canisters
+---
+
+![](/img/how-it-works/canister-lifecycle.600x300.jpg)
+
+# Canisters
+
+On the Internet Computer smart contracts come in the form of canisters.  These are computational units which bundle together code and state. Each canister defines a set of entry points which can be called by other canisters and parties external to the IC, such as browsers or mobile apps.
+When deployed, canisters behave much like actors from the actor-based concurrency model – they communicate asynchronously with each other but, unlike in the traditional actor model, communication is bidirectional. 
```

Comment:
I think most readers will not have heard of the actor model – so it may be preferable to formulate this part without referring to that model. I like the way you use and introduce it in the "Go Deeper" section, though. But it would not seem appropriate to do the same here.

Reply:
Removed